### ISSUE TITLE
Experian agreement needs a header

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4073]
## Area or Component
Experian agreement in the sign-up flow

## Quick Summary
There's no header element on the Experian agreement page.

## Accessibility Violation
Two issues: "Identity verification consent" is acting as a header but is semantically a paragraph, and there's no H1 element on the page.

https://cdc-devtools.dequecloud.com/rule-help/en/semantic-heading
https://dequeuniversity.com/rules/axe/4.4/page-has-heading-one?application=AxeChrome

## Action Requested
Change "Identity verification consent" from a paragraph element to a header. The styling shouldn't change, but the DOM needs to.

## Screenshots (optional)
![Screen Shot 2022-07-26 at 2 55 57 PM](https://user-images.githubusercontent.com/80282552/181119319-eee995fa-1d8e-47a6-8ae8-a19d86b444bf.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=66:66
